src/components/TopTabs/index.tsx

```diff
@@ -0,0 +1,27 @@
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import styles from './styles.module.css';
+
+type Tab = {
+  title: string;
+  url: string;
+};
+
+type Props = {
+  tabs: Tab[];
+};
+
+export default function TopTabs({ tabs }: Props) {
+  return (
+    <div className="flex alignCenter">
```

Comment:
can we use outlined tabs here
https://mantine.dev/core/tabs/

Reply:
Nice. I will try that.